.github/workflows/rust-threshold-schnorr-example.yml

```diff
@@ -25,11 +25,13 @@ jobs:
         run: bash .github/workflows/provision-pocket-ic-server.sh
       - name: Rust Threshold Schnorr Darwin
         run: |
+          dfxvm default 0.22.0-beta.0
```

Comment:
Should we track removing this hardcoded version and re-enabling the disabled tests in a JIRA ticket?

Reply:
I don't know if I am doing anything wrong, but I if I follow the read me and try to sign I get `IC0302: Management canister has no method 'sign_with_schnorr`, which seems to indicate I am using the wrong version of dfx

Reply:
This indeed indicates that the dfx version is too old.

@altkdf, should we adapt the READMEs accordingly?

I guess the README should say that at least dfx version `0.22.0-beta.0` is required, while pointing to [this](https://github.com/dfinity/dfxvm) regarding installing a specific dfx version, or directly giving the respective command:
```
DFX_VERSION=0.22.0-beta.0 sh -ci "$(curl -fsSL https://internetcomputer.org/install.sh)"
```

Reply:
I added to both READMEs that at least `0.22.0-beta.0` is required and added a ticket in the cleanup (CRP-2545) to track tests.

Reply:
Re hard-coded version: I'm not sure if that's a problem TBH. I actually think it might be even better because that's what we do for rust e.g. as well and makes tests reproducible. What we are missing IMO is something like `dfx default '>=0.22.0-beta.0'` with a lock file, but nothing like that exists so far, right?

Reply:
> Re hard-coded version: I'm not sure if that's a problem TBH. I actually think it might be even better because that's what we do for rust e.g. as well and makes tests reproducible.

There is an [Examples Maintenance Process](https://docs.google.com/document/d/1nQA3ISsZBo9HnwsNrZO4GqpSL1nCDxUu1PjQ_Yt0Gyo/edit) that says the following.

> Any time a new version of DFX is released (beta versions included), a new PR will be raised against the examples repo that updates the DFX_VERSION variable. Specifically, the files changed will be [provision-darwin.sh](https://github.com/dfinity/examples/blob/master/.github/workflows/provision-darwin.sh) and [provision-linux.sh](https://github.com/dfinity/examples/blob/master/.github/workflows/provision-linux.sh).

If we were to hard-code a DFX-version, this maintenance process would no longer catch issues. So, removing the hardcoded version should be tracked in a ticket.

Reply:
Ah, makes sense, didn't know that! Created https://dfinity.atlassian.net/browse/CRP-2546.

Reply:
👍